Gemfile

```diff
@@ -1,30 +1,39 @@
+#This file is generated by ModuleSync, do not edit.
+
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
-group :development, :unit_tests do
-  gem 'rake',                    :require => false
-  gem 'rspec-puppet',            :require => false
-  gem 'puppetlabs_spec_helper',  :require => false
-  gem 'puppet-lint',             :require => false
-  gem 'simplecov',               :require => false
-  gem 'puppet_facts',            :require => false
-  gem 'json',                    :require => false
+def location_for(place, version = nil)
```

Comment:
it's coming from modulesync_configs; there is no real libraries for Gemfiles: Catch-22